### PR TITLE
fix(perf-latency-nemesis): Set large test duration

### PR DESCRIPTION
**Pull request by aleksbykov**
Incease test duration and stress write command duration up to 2 days to get email results

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: test-cases/performance/perf-regression-latency-650gb-with-nemesis.yaml [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
                     "cassandra-stress write no-warmup cl=ALL n=162500000 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=200 -col 'size=FIXED(128) n=FIXED(8)' -pop seq=325000000..487500000",
                     "cassandra-stress write no-warmup cl=ALL n=162500000 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=200 -col 'size=FIXED(128) n=FIXED(8)' -pop seq=487500000..650000000"]
 
-stress_cmd_w: "cassandra-stress write no-warmup cl=QUORUM duration=1600m -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate 'threads=250 fixed=20332/s' -col 'size=FIXED(128) n=FIXED(8)' -pop 'dist=gauss(1..650000000,325000000,9750000)' "
+stress_cmd_w: "cassandra-stress write no-warmup cl=QUORUM duration=2500m -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate 'threads=250 fixed=20332/s' -col 'size=FIXED(128) n=FIXED(8)' -pop 'dist=gauss(1..650000000,325000000,9750000)' "
```
Comment (fgelcer) [inline test-cases/performance/perf-regression-latency-650gb-with-nemesis.yaml:7]: shouldn't it be `duration=2850` to have load running during the entire duration? here there is a chance we will be running last operations without stress

Comment (aleksbykov) [reply test-cases/performance/perf-regression-latency-650gb-with-nemesis.yaml:7]: fixed

**Comment by aleksbykov** [comment]
@roydahan @fgelcer @roydahan review please

**Review by fgelcer — APPROVED** [review]
LGTM

**Review by roydahan — APPROVED** [review]

**Comment by roydahan** [comment]
There is a conflict to backport because of the change to "NetworkTopologyStrategy".
I don't know if we want to change that in v14 because it may cause a regression due to a test change that we're introducing here.

**Comment by fruch** [comment]
> There is a conflict to backport because of the change to "NetworkTopologyStrategy". I don't know if we want to change that in v14 because it may cause a regression due to a test change that we're introducing here.

@roydahan so what was the end of it ? we don't need this change anymore ?

also the header of this PR doesn't explain the *why*, so I can't even tell if it's important or not

**Comment by roydahan** [comment]
Maybe we backport it with a PR?
@aleksbykov didn't we need this? if not let's revert also from master.

**Pull request merged**